invokeai/app/invocations/image.py

```diff
@@ -118,6 +124,80 @@ def invoke(self, context: InvocationContext) -> ImageOutput:
             image=image,
         )
 
+class DownloadImageInvocation(BaseInvocation):
```

Comment:
This feels potentially risky as a node, but also useful. May want someone else's opinion on it.